### PR TITLE
Add syndication boxnumber table

### PR DESCRIPTION
**Pull request by emalfroy**

### CODE REVIEW

**Review by jvandaal — APPROVED** [review]

File: src/AddressRegistry.Projections.Legacy/AddressSyndication/AddressSyndicationExtensions.cs [hunk 1]
```diff
@@ -6,12 +6,14 @@ namespace AddressRegistry.Projections.Legacy.AddressSyndication
...
     using System.Threading.Tasks;
     using System.Xml;
+    using System.Xml.Schema;
     using Be.Vlaanderen.Basisregisters.EventHandling;
     using Be.Vlaanderen.Basisregisters.GrAr.Common;
     using Be.Vlaanderen.Basisregisters.GrAr.Provenance;
     using Be.Vlaanderen.Basisregisters.ProjectionHandling.Connector;
     using Be.Vlaanderen.Basisregisters.ProjectionHandling.SqlStreamStore;
     using Microsoft.EntityFrameworkCore;
+    using Microsoft.VisualBasic;
```
Comment (ArneD) [inline src/AddressRegistry.Projections.Legacy/AddressSyndication/AddressSyndicationExtensions.cs:16]: uhm?

File: src/AddressRegistry.Projections.Legacy/Migrations/20220919142202_AddAddressBoxNumberSyndicationHelper.cs [hunk 2]
```diff
@@ -0,0 +1,43 @@
+﻿using System;
```
Comment (lvermeulen) [inline src/AddressRegistry.Projections.Legacy/Migrations/20220919142202_AddAddressBoxNumberSyndicationHelper.cs:1]: move inside namespace

**Review by ArneD — CHANGES_REQUESTED** [review]

File: src/AddressRegistry.Projections.Legacy/Migrations/20220919142202_AddAddressBoxNumberSyndicationHelper.cs (hunk 2 above)
Comment (emalfroy) [reply src/AddressRegistry.Projections.Legacy/Migrations/20220919142202_AddAddressBoxNumberSyndicationHelper.cs]: Done

File: src/AddressRegistry.Projections.Legacy/AddressSyndication/AddressSyndicationExtensions.cs (hunk 1 above)
Comment (emalfroy) [reply src/AddressRegistry.Projections.Legacy/AddressSyndication/AddressSyndicationExtensions.cs]: removed

**Pull request merged**